### ISSUE TITLE
Baritone Refuses to Jump One Block Diagonally

### ISSUE THREAD

**Issue by Jebula999** [cabaletta/baritone#979]
Hi There,

I have come to an issue where if you set allowPlace and allowBreak to false, and you try to make Baritone pathfind along a diagonal gap, it will not jump.

When enabling allowPlace and allowBreak, it will then build in those areas instead of simply jumping.

What can be done to get Baritone to jump these gaps?

eg:
x=stone
o=air

xxoxx He will jump this no problem

xoo
ooo He will not jump this game at all
oox

**Comment by NavnDK** [comment]
yeah i have exact same issue i think it's becuse it's not designed to do parkour non the less i would VERY much love to have this added
